server/jetstream_test.go

```diff
@@ -88,12 +88,11 @@ func TestJetStreamBasicNilConfig(t *testing.T) {
 	}
 }
 
-func RunBasicJetStreamServer() *Server {
+func RunBasicJetStreamServer(t testing.TB) *Server {
```

Comment:
I think we need to leave this one like this and a have a different one that takes the TB interface, maybe:
```suggestion
func RunTestJetStreamServer(t testing.TB) *Server {
```
or a private one would work `runBasicJetStreamServer`

Reply:
The method I modified is defined in `jetstream_test.go`, I don't think anyone outside this module would be able to see such symbol, no?

Reply:
you are right, yes it would not have been exposed so would be be ok change